.travis.yml

```diff
@@ -58,6 +58,10 @@ env:
     TRAVIS_JDK=adopt@1.8.0-222
   - SCALA_VERSION=2.13.2
     TRAVIS_JDK=adopt@1.11.0-4
+  - SCALA_VERSION=2.13.3
+    TRAVIS_JDK=adopt@1.8.0-222
```

Comment:
`-222` for any particular reason?

Reply:
No, I just stuck with the same version as for the others.

Reply:
SGTM. Let's bump the JDK versions elsewhere.